packages/cli/src/common/processor.ts

```diff
@@ -13,7 +13,7 @@ import path from "path";
 
 export async function processProject(
   options: any,
-  command: "deploy" | "release",
+  command: "deploy" | "publish",
```

Comment:
did we change the dashboard url also? cause this will open /contracts/publish/...